src/index.html

```diff
@@ -40,57 +42,42 @@ <h1 class="header__splash--title">Wheel of Death</h1>
-            <li class="ul__player--two--name" id="ul__player--two--name--js">Player Two Name</li>
-            <li class="ul__player--two--score" id="ul__player--two--score--js">Round $ <span class="span__player--2--score"></span></li>
-            <li class="ul__player--2--total--score" id="div__player--two--total--score--js">Game $ </p>
-            <li class="ul__player--three--name" id="ul__player--three--name--js">Player Three Name</li>
-            <li class="ul__player--three--score" id="ul__player--three--score--js">Round $ <span class="span__player--3--score"></span></li>
-            <li class="ul__player--3--total--score" id="div__player--three--total--score--js">Game $ </p>
-          </ul>
-          <p><span class="span__player--turn" id="span__player--turn--js"></span> it's your turn!</p>
-          <div class='guess__input--div'>
-            <button class='guess__btn--vowel' id='guess__btn--vowel--js' disabled>Buy A Vowel</button>
-            <button class='btn__spin' id='btn__spin--js'>Spin The Wheel</button>
-            <input class='guess__input' id='guess__input--js' placeholder="Guess Answer Here!">
-            <button class='guess__input--btn' id='guess__input--btn--js'>Submit Guess</button>
-            <p>Round <span class="span__round" id="span__round--js">1</span></p>
-          </div>
-          <img class="img__pumpkin" src="/images/pumpkin.png" alt="A pumkin wheel">
-        </div>
-      </container>
-      <label class="wheel__output--label">Spin Result: </label>
-      <p class="wheel__output" id="wheel__output--js"></p>
+        <ul class="container__ul--player--info">
+          <li class="ul__player--one--name" id="ul__player--one--name--js">Player One Name</li>
+          <li class="ul__player--one--score" id="ul__player--one--score--js">Round $ <span class="span__player--1--score"></span></li>
+          <li class="ul__player--1--total--score" id="div__player--one--total--score--js">Game $ </p>
+          <li class="ul__player--two--name" id="ul__player--two--name--js">Player Two Name</li>
+          <li class="ul__player--two--score" id="ul__player--two--score--js">Round $ <span class="span__player--2--score"></span></li>
+          <li class="ul__player--2--total--score" id="div__player--two--total--score--js">Game $ </p>
+          <li class="ul__player--three--name" id="ul__player--three--name--js">Player Three Name</li>
+          <li class="ul__player--three--score" id="ul__player--three--score--js">Round $ <span class="span__player--3--score"></span></li>
+          <li class="ul__player--3--total--score" id="div__player--three--total--score--js">Game $ </p>
+        </ul>
+    </container>
+      <!-- <div class="center_game-box"> -->
       <container class="main__container--puzzle">
           <!-- we could consider just disabling the buttons for the vowels/consonants that have been guessed instead of this area for letters guessed-->
```

Comment:
Can we remove the comments that aren't related to the structure of the page in this file before we merge it? 
Line 66 and 123